usecases/config/environment.go

```diff
@@ -769,5 +769,19 @@ func parseClusterConfig() (cluster.Config, error) {
 
 	cfg.FastFailureDetection = entcfg.Enabled(os.Getenv("FAST_FAILURE_DETECTION"))
 
+	// MAINTENANCE_NODES is experimental and subject to removal/change. It is an optional, comma
```

Comment:
**question (non-blocking):** what is the benefits of making it's , separated string includes multiple nodes, given that we just check only for the current node in our logic above 💭  ?

if the usage will be only for the current node, i'd say we convert it to bool or enum string instead of slice

Reply:
The idea is that it's a pain to set env vars on individual pods given our current k8s/helm setup and is preferred to be able to modify all the pods' env vars the same way. Here's a message from Etienne: "are you also planning a version with MAINTENANCE_MODE_HOSTS=weaviate-3 or something that can safely be rolled out to all nodes, but only affect one? for now I think it’s fine with the =true as long as we make sure that no other node can ever get this config. But this will be difficult to keep as we want to roll out future changes."